input/fsh/mcsd_organization.fsh

```diff
@@ -38,15 +46,31 @@ Title:    "LDAP schema"
 * contact.telecom -> "HCRegulatedOrganization.hpdMedicalRecordsDeliveryEmailAddress"
 
 
+Instance: ChmCSDOrganizationCommunityA
+InstanceOf: CHmCSDOrganization
+Title: "CH mCSD Organization Community A"
+Description: "An example of CHmCSDOrganization that contains the same information as Community A in the Swiss examples
```

Comment:
can we add a note in the description:

A communities Care Services Selective Supplier is not obliged to return the community itself as an entry, this is optional.

or should we add this information somewhere else?